go/install/update_test.go

```diff
@@ -12,16 +12,24 @@ import (
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/logger"
 	keybase1 "github.com/keybase/client/go/protocol"
+	"golang.org/x/net/context"
 )
 
 func NewTestUpdater(t *testing.T, config keybase1.UpdateConfig) Updater {
 	context := libkb.NewGlobalContext()
+	context.Init()
 	context.Log = logger.NewTestLogger(t)
 	return NewUpdater(context, config, testUpdateSource{})
 }
 
 type testUpdateSource struct{}
 
+type nullUpdateUI struct{}
```

Comment:
we've been using this elsewhere, but you're right, maybe it's not the right word.